curator/patterns/post-hero.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Title: Single Post Hero
+ * Slug: curator/post-hero
+ * Inserter: no
+ */
+
+?>
+
+<?php if ( has_post_thumbnail() ): ?>
```

Comment:
I would avoid this logic inside the patterns, I think this should be handled by the cover block itself. In other words, the block should be able to detect if there is a featured image, and render itself accordingly. 

My recommendation for now is to remove it and use the "useFeaturedImage" version of the cover block you have here.

Reply:
Noting that the tradeoff with this recommendation, I think for now we have to be okay with the following state if there is no featured image set: 

<img width="1904" alt="Screen Shot 2022-05-23 at 11 22 53 AM" src="https://user-images.githubusercontent.com/5375500/169854947-6ccf28e6-e0f7-4f88-8e66-3eb6b02f4a04.png">

Reply:
Got u @jffng , I'll adopt your recommendation.